### PR TITLE
Change all 'flask.ext.login' references to 'flask_login'

### LINKED ISSUE #282: ExtDeprecationWarning: Importing flask.ext.login is deprecated, use flask_login instead.
When I start an application with the newest flask version (0.11), I get the warning:

```
/usr/lib/python3.5/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.login is deprecated, use flask_login instead.
```
Please update the docs to match that.

### PR DESCRIPTION
**Pull request by joeyespo**
This fixes #282.

See https://github.com/pallets/flask/pull/1484 for more information.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ function. For example::
             flask.flash('Logged in successfully.')
 
             next = flask.request.args.get('next')
-            # next_is_valid should check if the user has valid 
+            # next_is_valid should check if the user has valid
```
Comment (maxcountryman) [inline docs/index.rst]: There's a lot changed in this file that doesn't seem relevant to this patch. It makes it difficult to evaluate the patch on its own and I don't generally think it's good practice to merge commits with irrelevant changes.

**Comment by alanhamlett** [comment]
Merged with #289

File: docs/index.rst (hunk 1 above)
Comment (joeyespo) [inline docs/index.rst]: Agreed. I should have reviewed better before PR'ing.

**Comment by joeyespo** [comment]
Thanks, @alanhamlett.
